testdata/baselines/reference/submodule/conformance/commonJSAliasedExport.js.diff

```diff
@@ -10,13 +10,12 @@
  function funky(declaration) {
      return false;
  }
-+export = donkey;
  module.exports = donkey;
 +export var funky = funky;
```

Comment:
Not your bug but definitely an interesting one.

Reply:
Yes, right, I was just commenting on the remaining diff in the file.

Reply:
Got it. Definitely funky!